include/nil/blueprint/handle_component.hpp

```diff
@@ -66,11 +66,28 @@
 namespace nil {
     namespace blueprint {
 
+        enum class generate_flag : uint8_t {
```

Comment:
The name seems confusing to me. Let's maybe consider something like "assigner_mode"?

Reply:
`assigner_mode::ASSIGNER` - are you sure?

Reply:
Maybe "generation_mode" then?

Reply:
yes, sound good